src/main/scala/com/softwaremill/tapir/scalaxb/example/Endpoints.scala

```diff
@@ -0,0 +1,42 @@
+package com.softwaremill.tapir.scalaxb.example
+
+import sttp.tapir.{PublicEndpoint, Schema, endpoint}
+import cats.effect.IO
+import generated.Outer
+import sttp.tapir.docs.apispec.DocsExtensionAttribute.RichSchema
+import sttp.tapir.generic.auto._
+import sttp.tapir.generic.Derived
+import sttp.tapir.json.circe._
+import io.circe.generic.auto._
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.swagger.bundle.SwaggerInterpreter
+
+object Endpoints {
+
+  // updated `Outer` schema with custom docs extension, so that example bodies at Swagger would have `xmlns`
+  // attribute (look at "Prefixes and Namespaces" section at https://swagger.io/docs/specification/data-models/representing-xml/),
+  // that is needed by scalaxb code to property decode given xml
+  case class XmlNamespace(namespace: String)
+  implicit val outerSchemaWithXmlNamspace: Schema[Outer] = implicitly[Derived[Schema[Outer]]].value
```

Comment:
since we're in Scala3 we can use a `given` without a name

Reply:
Unfortunately, we are not. Scalaxb would work only on Scala 2.11, 2.12, and 2.13. This project uses Scala 2.13.8.

Reply:
ah ok I thought I saw scala3 in build.sbt :)